utils/toTitleCase.js

```diff
@@ -0,0 +1,12 @@
+// Taken from https://stackoverflow.com/a/27441234
+function toTitleCase(str){
```

Comment:
I'm not sure that we need all this functionality - we likely only need a function that can split `lisp-case` words. 
Tailwind has text transformation that can capitalise the words for you: https://tailwindcss.com/docs/text-transform

Reply:
Sure, we can change it to only handle `lisp-case`. I like the function capitalising the words though because it is unit tested whereas the `SiteNavigation` is not.